crates/red_knot_python_semantic/resources/mdtest/comparison/tuples.md

```diff
@@ -189,10 +186,8 @@ c = (1, 2, 3)
 reveal_type(a is (1, 2))  # revealed: bool
 reveal_type(a is not (1, 2))  # revealed: bool
 
-# TODO: Update to Literal[False] once str == int comparison is implemented
-reveal_type(a is b)  # revealed: @Todo
-# TODO: Update to Literal[True] once str == int comparison is implemented
-reveal_type(a is not b)  # revealed: @Todo
```

Comment:
Same issue as mentioned in the previous comment

Reply:
Here too I think all we need is the explicit handling of mismatched literals case, this should never fallback to `Type::Instance` and typeshed at all, because the types are totally literal -- we just don't have the required cases in the match statement yet